kge/job/train.py

```diff
@@ -246,45 +252,55 @@ def save(self, filename) -> None:
             filename,
         )
 
-    def load(self, filename: str) -> str:
+    def load(self, checkpoint: Dict, model: Optional[KgeModel] = None) -> str:
```

Comment:
Hmm. Perhaps we should use "load" for instance methods and "create" / "create_from" for class methods? Would this make it clearer?

Reply:
done